app/helpers/indoor_helper.rb

```diff
@@ -17,6 +17,8 @@ def build_room_from(room_node, building)
 
     points.push(points.first)
     outer_shape = Polyline.create(points: points)
-    Room.create(building: building, outer_shape: outer_shape, full_name: name)
+
+    floor = Floor.create(name: 'Example Floor', building: building)
```

Comment:
Remove this for now, floors aren't being parsed yet. Mocking it may lead to confusion